submissions/SofiiaTrokhymchuk/pop-up-task/css/style.css

```diff
@@ -0,0 +1,217 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    text-decoration: none;
+    list-style-type: none;
+}
+
+body {
+    font-family: Verdana, Tahoma, sans-serif;
+    color: #212121;
+    background: #e6e6e6;
+    font-size: 1.2rem;
+    height: 100vh;
+}
+
+a:link,
+a:visited {
+    color: inherit;
+}
+
+img[src$=".svg"] {
+    max-height: 30px;
+}
+
+.container {
+    padding: 15px 50px;
+    max-width: 1000px;
+    margin: 0 auto;
+    position: relative;
+}
+
+.header {
+    display: grid;
+    align-items: center;
+    grid-template-columns: 1fr 2fr;
+    padding: 15px 50px;
```

Comment:
what is this trick for? :) I mean padding and margin

Reply:
I was playing with paddings and margins inside a container div, and forgot to delete it =(

Reply:
I was considering a `.container` as a `div` for all the content on the page. however, I didn't want the `header` to have that padding as well, that's why I used negative margin. but I also want my `header` to have the same padding as a `.container`. without these properties the `header` looks collapsed